### DISCUSSION TITLE
Good way to deal with index.html when I can´t use /

### DISCUSSION THREAD

**Discussion by tovare** [observablehq/framework#1680] [Q&A] [answered]
Hi,
I need to stop Observable Framework from changing URLs because I am on a server which cannot deal properly with redirecting / to /index.html and I cannot configure.

The way currently dealt with this is to:

1. turn off the side-menu
2. navigation on the bottom.
3. Implement an index-page whth a link to an index2 page (This allows me to link to index2 in the header).

Any ideas ?

**Comment by mbostock** [comment]
Did you try setting the [`cleanUrls: false` config option](https://observablehq.com/framework/config#clean-urls)?

**Comment by tovare** [comment]
Yes, cleanUrls works well for all URLs not called index.html

"Regardless of this setting, a link to an index page will drop the implied index.html; for example foo/index.html will be formatted as foo/."

**Reply by mbostock** [reply, marked as answer]
Okay, sounds like we need another option like `cleanUrls: "preserve-index"` to support this.
